oci/private/load.bzl

```diff
@@ -218,7 +218,6 @@ def _load_impl(ctx):
 
     return [
         DefaultInfo(
-            files = depset([mtree_spec]),
```

Comment:
Is there any reason to consider adding another output group to reference output?

Reply:
I don't see any benefit for now. That can be added later as a feature if anybody asks for it